packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -17,6 +17,14 @@
   background: var(--atomic-primary-background);
 }
 
+.placeholder-wrapper {
+  height: 400px;
```

Comment:
Maybe we'd want this to be overridable by an implementer. 400px might be too much for a widget like IPX.

Reply:
The answer tho might still be 400px tall, and so the issue might repeat itself I reckon.
@nathanlb, I propose that we see if this is actually a problem on the field before adding an option.
Thought?

Reply:
Looking at the demo, when the answer goes beyond the defined height, it still waits to generate completely before pushing everything at once. I tend to agree with @nathanlb and want it to be overridable by implementer because 400px is very big

Reply:
exposed it so it can be overridable via css